app/components/ProfileHeader/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import PageSection from 'components/PageSection';
+import BorderBottomDiv from 'components/shared/BorderBottomDiv';
+
+// Easily keeps image/logo within good size range.
+const ProfileImg = styled.img`
+  min-height: 75px;
+  min-width: 75px;
+  max-height: 100px;
+  max-width: 100px;
+`
+
+const ProfileHeader = (props) => (
+  <div className="text-center">
+    <BorderBottomDiv>
+      <ProfileImg src={props.url} className="mt-4"
+        alt={props.guide ? 'guide logo' :'profile image'}
+      />
+      <h1 className="h2 text-info">{props.name}</h1>
+      {
+        // Only creates a location/summary element if one exists
+        (props.location || props.summary) &&
+        <h2 className="h5 mb-3 text-muted">{props.location ? props.location : props.summary}</h2>
```

Comment:
h5 classname? What's the point of that?

Reply:
Font sizing (see previous comment). My logic here is that the heading is significant enough to be an h2 (rather important), but since the mock-up showed a smaller font, it was simpler to use Bootstrap's h5 class (as recommended) rather than make a new styled component.